cli/crates/server/src/errors.rs

```diff
@@ -142,8 +142,8 @@ pub enum ServerError {
     #[error("could not read the project database directory\nCaused by: {0}")]
     ReadDatabaseDir(IoError),
 
-    /// returned if an available port cannot be found for the bridge server
-    #[error("could not find an available port for the bridge server")]
+    /// returned if an available port cannot be found for the bridge server or playground
+    #[error("could not find an available port for an internal server")]
```

Comment:
OPT: IMHO "could not find an available port" would have been enough.

Reply:
That could be confused with the main port for the worker

Reply:
is there a difference though for the user? Or do we need that for ourselves for investigation?

Reply:
It signals that it's not related to the port search you passed, e.g. if you chose a really large port